backend/server.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
```

Comment:
lest get perm storage for sessions into a db - otherwise,

1. when the server breaks, people are logged out
2. most websites are run on multiple serves... servers dont share memory. that means from one visit to another i might be logged in or logged out - who knows?